packages/nimble-components/src/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -156,22 +156,30 @@ export class RichTextEditorPageObject {
             .map(el => el.textContent || '');
     }
 
-    private getEditorSection(): Element | null | undefined {
-        return this.richTextEditorElement.shadowRoot?.querySelector('.editor');
+    public getFormattingButton(
+        index: ToolbarButton
+    ): ToggleButton | null | undefined {
+        const buttons: NodeListOf<ToggleButton> = this.richTextEditorElement.shadowRoot!.querySelectorAll(
+            'nimble-toggle-button'
+        );
+        return buttons[index];
+    }
+
+    public getActiveElementFromDocument(): Element | null {
```

Comment:
Let's try to avoid exposing `Element` types on the public API of the page object since that exposes info about the DOM structure of the component, which should be an internal implementation detail. Instead let's expose methods like 
```ts
isRichTextEditorActiveElement(): boolean {
    return 
        document.activeElement === this.richTextEditorElement &&
        document.activeElement?.shadowRoot?.activeElement === this.getTiptapEditor();
}
```

(or those two checks could be split into separate public methods if needed)